src/ImageSharp.Drawing/project.json

```diff
@@ -0,0 +1,82 @@
+{
+  "version": "1.0.0-alpha1-*",
+  "title": "ImageSharp.Drawing",
+  "description": "A cross-platform library for the processing of image files; written in C#",
+  "authors": [
+    "James Jackson-South and contributors"
+  ],
+  "packOptions": {
+    "owners": [
+      "James Jackson-South and contributors"
+    ],
+    "projectUrl": "https://github.com/JimBobSquarePants/ImageSharp",
+    "licenseUrl": "http://www.apache.org/licenses/LICENSE-2.0",
+    "iconUrl": "https://raw.githubusercontent.com/JimBobSquarePants/ImageSharp/master/build/icons/imagesharp-logo-128.png",
+    "requireLicenseAcceptance": false,
+    "repository": {
+      "type": "git",
+      "url": "https://github.com/JimBobSquarePants/ImageSharp"
+    },
+    "tags": [
+      "Image Resize Crop Gif Jpg Jpeg Bitmap Png Core"
+    ]
+  },
+  "buildOptions": {
+    "allowUnsafe": true,
+    "xmlDoc": true,
+    "additionalArguments": [
+      "/additionalfile:stylecop.json"
+    ]
+  },
+  "configurations": {
+    "Release": {
+      "buildOptions": {
+        "warningsAsErrors": true,
+        "optimize": true
+      }
+    }
+  },
+  "dependencies": {
+    "ImageSharp": {
+      "target": "project",
+      "version": "1.0.0-*"
+    },
+    "ImageSharp.Processing": {
+      "target": "project",
+      "version": "1.0.0-*"
+    },
+    "StyleCop.Analyzers": {
+      "version": "1.0.0",
+      "type": "build"
+    },
+    "System.Buffers": "4.0.0",
+    "System.Numerics.Vectors": "4.1.1",
+    "System.Runtime.CompilerServices.Unsafe": "4.0.0"
+  },
+  "frameworks": {
+    "netstandard1.1": {
+      "dependencies": {
+        "System.Collections": "4.0.11",
```

Comment:
I also think we need to check if we still need all these dependencies. Maybe some of the projects no longer require all the dependencies?